docs/contributing-to-airbyte/updating-documentation.md

```diff
@@ -20,3 +20,90 @@ If you're adding new files, don't forget to update `docs/SUMMARY.md`.
 
 Once you're satisfied with your changes just follow the regular PR process.
 
+## Connector Documentation Best Practices
+Connectors typically have the following documentation elements: 
+
+* READMEs
+* Changelogs
+* Github Issues & Pull Requests
+* Source code comments
+* How-to guides
+
+Below are some best practices related to each of these. 
+
+### READMEs
+Every module should have a README containing:
+
+* A brief description of the module
+* development pre-requisites (like which language or binaries are required for development)
+* how to install dependencies
+* how to build and run the code locally & via Docker
+* any other information needed for local iteration
+  
+### Changelogs
```

Comment:
We should have people use this format for consistency:

## Changelog

```
| Version | Date       | Pull Request | Subject |
| :------ | :--------  | :-----       | :------ |
| 0.2.0   | 20XX-05-XX | [PR2#](https://github.com/airbytehq/airbyte/pull/PR2#) | description |
| 0.1.0   | 20XX-04-XX | [PR#](https://github.com/airbytehq/airbyte/pull/PR#) | description |
```

Reply:
@sherifnada Abhi and I are adding the changelog into the connector documentation.

Reply:
done